setup.py

```diff
@@ -62,7 +62,7 @@
 
 install_requires = [
     'Flask>=0.11.1',
-    'Flask-CeleryExt>=0.2.2',
+    'Flask-CeleryExt>=0.2.2,<0.3.0',
```

Comment:
We will need a solution for this, otherwise we are blocking usage of Celery 4. Let's talk about options IRL